aten/src/ATen/native/quantized/cpu/qnnpack_utils.h

```diff
@@ -221,3 +227,69 @@ Tensor qnnpack_avg_pool2d(
 } // namespace native
 } // namespace at
 #endif
+
+namespace {
+std::vector<float> generate_requantization_scales(
+    const at::Tensor& weight_scales,
+    const float input_scale,
+    const float output_scale) {
+  // Since weight scale is allocated with padding
+  // weight_scales.numel() gives us padded num elements.
+  auto num_output_channels_padded = weight_scales.numel();
+  float* weight_scales_data = weight_scales.data_ptr<float>();
+  std::vector<float> requant_scales(num_output_channels_padded, 1.f);
+  for (int i = 0; i < num_output_channels_padded; ++i) {
+    requant_scales[i] = weight_scales_data[i] * input_scale / output_scale;
```

Comment:
For aligning numerics with FBGEMM, we want to compute the reciprocal of the output scale first:
inv_C_scale_fp32 = 1.0f / C_scale_fp32.
https://fb.quip.com/nsJmAabU2Bas

Reply:
@raghuramank100, sure. Sorry I missed that. But that would have to be done in a separate PR.